.github/workflows/create_release.yml

```diff
@@ -5,141 +5,157 @@
 name: Create Releases
 on:
   schedule:
-    # Run weekly on Monday 00:00 UTC 
+    # Run weekly on Monday 00:00 UTC
     - cron:  '00 00 * * MON'
   push:
     branches: [main, rel-*]
   pull_request:
     branches: [main, rel-*]
-    types: 
+    types:
       - labeled  # Trigger when a label is added to a PR, more information: https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request
-  workflow_dispatch:    
+  workflow_dispatch:
```

Comment:
We need to ensure it can only be triggered on the main branch

Reply:
I think we need to make sure that only main-branch wheels can be published

Reply:
We check for that: (github.ref == 'refs/heads/main')

Reply:
Thanks!